runtime/moonbase/src/lib.rs

```diff
@@ -638,13 +662,15 @@ impl pallet_identity::Config for Runtime {
 	type MaxSuffixLength = MaxSuffixLength;
 	type MaxUsernameLength = MaxUsernameLength;
 	type WeightInfo = moonbase_weights::pallet_identity::WeightInfo<Runtime>;
+	type UsernameDeposit = ();
```

Comment:
We want something like:
```suggestion
	type UsernameDeposit = ConstU128<{ currency::deposit(0, MaxUsernameLength::get()) }>;
```